source/apiVolontaria/volunteer/views.py

```diff
@@ -342,6 +345,76 @@ def get_queryset(self):
             queryset = queryset.filter(user__username=username)
         return queryset
 
+    def generate_email_invitation(self, obj):
+        cal = Calendar()
+
+        cal.add('prodid', '-//{0} Events Calendar//'.format(obj.event.task_type.name, ))
+        cal.add('version', '2.0')
+
+        ical_event = ICalEvent()
+        ical_event.add('summary', obj.event.task_type.name)
+        ical_event.add('dtstart', obj.event.start_date)
+        ical_event.add('dtend', obj.event.end_date)
+        ical_event.add('dtstamp', obj.event.end_date)
+        ical_event['uid'] = str(obj.id)
+
+        cal.add_component(ical_event)
+        calendar = cal.to_ical()
+
+        #import base64
+        #calendar_encoded = base64.b64encode(b'%s' % calendar)
+
+        print(calendar)
+
+        if settings.CONSTANT['EMAIL_SERVICE'] is True:
+            MAIL_SERVICE = settings.SETTINGS_IMAILING
+
+            # Send email with a SETTINGS_IMAILING
+            from mailin import Mailin
+            m = Mailin("https://api.sendinblue.com/v2.0", MAIL_SERVICE["API_KEY"])
+            
+            data = {
+                "id": MAIL_SERVICE["TEMPLATES"]["CALENDAR_INVITATION"],
+                "to": "yanic.olivier@gmail.com",
```

Comment:
Please do not forget to use only settings variable for this kind of value

Reply:
Of course, it's really a WIP ;)